control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -142,6 +142,7 @@ func ReconcileKubeAPIServerDeployment(deployment *appsv1.Deployment,
 			SchedulerName:                 corev1.DefaultSchedulerName,
 			AutomountServiceAccountToken:  pointer.BoolPtr(false),
 			InitContainers: []corev1.Container{
+				util.BuildContainer(kasContainerWaitForEtcd(), buildKASContainerWaitForEtcd(images.CLI, deployment.Namespace)),
 				util.BuildContainer(kasContainerBootstrap(), buildKASContainerBootstrap(images.ClusterConfigOperator)),
```

Comment:
what is this other init container?

Reply:
that is for extracting config manifests from the configoperator image, they need to be applied async to the apiserver otherwise the apiserver never becomes available.